<h1>Validation FAILED.</h1><p>Build Time: 9 minutes 45 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-portal-workflow/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/com-liferay-portal-workflow/commit/54d654346489d47ff8ea556cacc8315ee96c5d35">54d654346489d47ff8ea556cacc8315ee96c5d35</a></p><h4>Task Summary:</h4><ul><li><strong><a href="https://test-1-6.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/161/subrepository-compile-jdk8.log">subrepository-compile-jdk8</a></strong> -  :white_check_mark:</li><li><strong><a href="https://test-1-6.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/161/subrepository-semantic-versioning-jdk8.log">subrepository-semantic-versioning-jdk8</a></strong> -  :white_check_mark:</li><li><strong><a href="https://test-1-6.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/161/subrepository-pmd-jdk8.log">subrepository-pmd-jdk8</a></strong> -  :white_check_mark:</li><li><strong><a href="https://test-1-6.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/161/subrepository-source-format-jdk8.log">subrepository-source-format-jdk8</a></strong> -  :x:<pre><code>     [echo] Using proxy squid.lax.liferay.com:3128.
     [echo] Configuration on demand is an incubating feature.
     [echo] :apps:forms-and-workflow:portal-workflow:checkSourceFormatting
     [echo] /opt/dev/projects/github/liferay-portal/modules/apps/forms-and-workflow/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/WorkflowGroupServiceSettings.java
     [echo] :apps:forms-and-workflow:portal-workflow:checkSourceFormatting FAILED
     [echo] 
     [echo] BUILD FAILED
     [echo] 
     [echo] Total time: 20.868 secs
     [echo] Gradle build finished at 2017-10-20 23:16:59.192.
     [echo] Exception in thread "main" com.liferay.source.formatter.SourceMismatchException: ./portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/WorkflowGroupServiceSettings.java expected:<...tal.kernel.settings.[Settings;
     [echo] import com.liferay.portal.kernel.settings.SettingsFactoryUtil;
     [echo] import com.liferay.portal.kernel.settings.TypedSettings];
     [echo] import com.liferay...> but was:<...tal.kernel.settings.[GroupServiceSettingsLocator;
     [echo] import com.liferay.portal.kernel.settings.Settings;
     [echo] import com.liferay.portal.kernel.settings.SettingsFactoryUtil;
     [echo] import com.liferay.portal.kernel.settings.TypedSettings;
     [echo] import com.liferay.portal.kernel.util.PortalUtil];
     [echo] import com.liferay...>
     [echo] 	at com.liferay.source.formatter.BaseSourceProcessor.processFormattedFile(BaseSourceProcessor.java:387)
     [echo] 	at com.liferay.source.formatter.JavaSourceProcessor.format(JavaSourceProcessor.java:77)
     [echo] 	at com.liferay.source.formatter.BaseSourceProcessor._format(BaseSourceProcessor.java:505)
     [echo] 	at com.liferay.source.formatter.BaseSourceProcessor._performTask(BaseSourceProcessor.java:608)
     [echo] 	at com.liferay.source.formatter.BaseSourceProcessor.access$000(BaseSourceProcessor.java:68)
     [echo] 	at com.liferay.source.formatter.BaseSourceProcessor$1.call(BaseSourceProcessor.java:114)
     [echo] 	at com.liferay.source.formatter.BaseSourceProcessor$1.call(BaseSourceProcessor.java:110)</code></pre></li><li><strong><a href="https://test-1-6.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(master)/builds/161/subrepository-unit-jdk8.log">subrepository-unit-jdk8</a></strong> -  :white_check_mark:</li></ul><h5>For more details click <a href="https://test-1-21.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(master)/148/">here</a>.</h5>

--- a/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/WorkflowGroupServiceSettings.java
+++ b/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/WorkflowGroupServiceSettings.java
@@ -15,8 +15,8 @@
 package com.liferay.portal.workflow.kaleo.runtime.internal;
 
 import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.settings.CompanyServiceSettingsLocator;
 import com.liferay.portal.kernel.settings.FallbackKeys;
-import com.liferay.portal.kernel.settings.GroupServiceSettingsLocator;
 import com.liferay.portal.kernel.settings.Settings;
 import com.liferay.portal.kernel.settings.SettingsFactoryUtil;
 import com.liferay.portal.kernel.settings.TypedSettings;
@@ -30,12 +30,12 @@ import com.liferay.portal.kernel.workflow.WorkflowConstants;
 @Settings.Config(settingsIds = WorkflowConstants.SERVICE_NAME)
 public class WorkflowGroupServiceSettings {
 
-	public static WorkflowGroupServiceSettings getInstance(long groupId)
+	public static WorkflowGroupServiceSettings getInstance(long companyId)
 		throws PortalException {
 
 		Settings settings = SettingsFactoryUtil.getSettings(
-			new GroupServiceSettingsLocator(
-				groupId, WorkflowConstants.SERVICE_NAME));
+			new CompanyServiceSettingsLocator(
+				companyId, WorkflowConstants.SERVICE_NAME));
 
 		return new WorkflowGroupServiceSettings(settings);
 	}

--- a/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/notification/EmailNotificationSender.java
+++ b/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/notification/EmailNotificationSender.java
@@ -71,11 +71,11 @@ public class EmailNotificationSender
 		Map<String, Serializable> workflowContext =
 			executionContext.getWorkflowContext();
 
-		long groupId = GetterUtil.getLong(
-			workflowContext.get(WorkflowConstants.CONTEXT_GROUP_ID));
+		long companyId = GetterUtil.getLong(
+			workflowContext.get(WorkflowConstants.CONTEXT_COMPANY_ID));
 
 		WorkflowGroupServiceSettings workflowGroupServiceSettings =
-			WorkflowGroupServiceSettings.getInstance(groupId);
+			WorkflowGroupServiceSettings.getInstance(companyId);
 
 		String fromAddress = (String)workflowContext.get(
 			WorkflowConstants.CONTEXT_NOTIFICATION_SENDER_ADDRESS);

